### DISCUSSION TITLE
WETH from metamask to Terra no value

### DISCUSSION THREAD

**Discussion by Donati75** [wormhole-foundation/wormhole#1567] [Developer Q&A] [answered]
Hello

i have deposit Wrapped Ethereum (WETH) from my Metamask to my terra wallet by using https://portalbridge.com to exchange WETH to LUNA on Terra Network

I have correctly receive and show my 1 WETH on /station.terra.money/wallet but when i want exchange 1 weth  (1700$ at this time) to LUNA, i get ≈ 4.547757 (1LUNA =6$ at this time). 

Who can i exchange correctly or recover my WETH on ethreum network ([https://portalbridge.com](https://t.co/kmccFXqYl5) don't show me WETH TOKEN to swap them from terra network to eth network)

Thanks

**Comment by evan-gray** [comment, marked as answer]
If this was Terra Classic, select the address `terra14tl83xcwqjy0ken9peu4pjjuu755lrry2uy25r`. If it was Terra 2, select `terra15hhqg8gyz04zapynqtk7uvlsp7lzay7etrt9ann0276v94yae63sxygeat`. These addresses were calculated with https://www.portalbridge.com/#/token-origin-verifier
